catalog/transformer/catalog-transformer-bootflag/src/main/java/org/codice/ddf/catalog/transformer/bootflag/InputTransformerBootServiceFlag.java

```diff
@@ -49,7 +49,7 @@
 
   private static final long DEFAULT_TRANSFORMER_WAIT_TIMEOUT_MILLIS = TimeUnit.MINUTES.toMillis(5);
 
-  private static final long DEFAULT_TRANSFORMER_CHECK_PERIOD_MILLIS = TimeUnit.SECONDS.toMillis(20);
+  private static final long DEFAULT_TRANSFORMER_CHECK_PERIOD_MILLIS = TimeUnit.MILLISECONDS.toMillis(500);
```

Comment:
✏️ Can just set to 500 since var name provides unit context.

Reply:
Yes